layouts/_default/single.html

```
@@ -0,0 +1,119 @@
+      {{ end }}
+    </div>
+
+    {{ if and (or (eq .Type "post") (eq .Type .Site.Params.postSectionName)) (ne .Page.Params.disableComments true) }}
+      {{- if .Site.Config.Services.Disqus.Shortname -}}
+        <div id="comment">
+          <h2>{{ i18n "comments" }}</h2>
+          {{ template "_internal/disqus.html" . }}
+        </div>
+      {{- end -}}
+      {{- if .Site.Params.utterances.repo -}}
+        <div id="comment">
```

HTMLHint (1.9.2)

[error] The id value [ comment ] must be unique.

(id-unique)